app/_data/docs_nav_ee_1.3-x.yml

```diff
@@ -277,6 +277,8 @@
           url: /plugins/oidc-auth0
     - text: GraphQL
       items:
+        - text: GraphQL Quickstart
```

Comment:
@GayleNeumann is this title okay in the navigation? On the page itself, the title is "Getting Started with GraphQL and Kong Enterprise"

Reply:
(Confirmed IRL that this works.)